src/utils/webvtt-parser.js

```
@@ -149,11 +153,8 @@ const WebVTTParser = {
             localTime = cueString2millis(cueTime) / 1000;
             // Convert MPEGTS to seconds from 90kHz.
             presentationTime = mpegTs / 90000;
-
-            if (localTime === -1) {
-              parsingError = new Error(`Malformed X-TIMESTAMP-MAP: ${line}`);
-            }
           } catch (e) {
+            timestampMap = false;
             parsingError = new Error(`Malformed X-TIMESTAMP-MAP: ${line}`);
```

Why not rethrow the error instead of making it again? You're never using the `Malformed X-TIMESTAMP-MAP: Local:${timeString});` thrown in `cueStringToMillis`.